library.c

```diff
@@ -434,7 +434,7 @@ redis_sock_read_scan_reply(INTERNAL_FUNCTION_PARAMETERS, RedisSock *redis_sock,
     }
 
     /* Push the iterator out to the caller */
-    *iter = atol(p_iter);
+    *cursor = strtoull(p_iter, NULL, 10);
```

Comment:
```suggestion
    *cursor = ZEND_STRTOUL(p_iter, NULL, 10);
```

Reply:
We can't use `ZEND_STRTOUL` here

On a 32-bit system:
```
ZEND_STRTOUL(4890909195324358656) = 4294967295
strtoull(4890909195324358656) = 4890909195324358656
```

Reply:
I guess windows doesn't have `strtoull` that's why I suggested to use zend wrapper but ok, nobody cares about windows :smile:

Reply:
That's actually a good point.  I checked to make sure that Windows had it but maybe I misunderstood and thought it did.  If we have a build issue I can fix it with a custom `#ifdef`.